bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs

```diff
@@ -189,13 +222,27 @@ public void Test_EncryptMediator_Hashes_Dont_Match()
                 var mediator = new EncryptionMediator(internalManifest, context, device);
                 Assert.IsNull(mediator);    // should not be created, so null at best
             }
-            catch (Exception ex)
+            catch (Exception)
             {
                 // if there is an exception then the manifest hash would not be equal
                 Assert.AreNotEqual(context.ManifestHash.ToHex(), internalManifest.ManifestHash.ToHex());
             }
         }
 
 
+        private static CiphertextBallotContest FindContestOrDefault(CiphertextBallot ballot, Func<CiphertextBallotContest, bool> func)
```

Comment:
I was tempted to put this into the Encryption project, but wasn't sure if we're trying to keep that clean

Reply:
This would be a great enhancement for v2.